RF24Gateway.cpp

```diff
@@ -667,7 +638,7 @@ void ESBGateway<mesh_t, network_t, radio_t>::handleTX()
 
     if (msg->size > MAX_PAYLOAD_SIZE)
     {
-        //printf("*****WTF OVER *****");
+        // printf("*****WTF OVER *****");
```

Comment:
# 🤣 
"WTF OVER" sounds like something I would add after 30 minutes of debugging.